osu.Game.Rulesets.Catch/Mods/CatchModNoScope.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mods;
+using osu.Framework.Utils;
+using osu.Game.Rulesets.Catch.Objects;
+using osu.Game.Rulesets.Catch.UI;
+using osu.Game.Rulesets.UI;
+using osu.Game.Utils;
+
+namespace osu.Game.Rulesets.Catch.Mods
+{
+    public class CatchModNoScope : ModNoScope, IApplicableToBeatmap, IApplicableToDrawableRuleset<CatchHitObject>
+    {
+        public override string Description => "Where's the catcher?";
+
+        public PeriodTracker BananaShowerPeriods;
+
+        public void ApplyToBeatmap(IBeatmap beatmap)
+        {
+            BananaShowerPeriods = new PeriodTracker(beatmap.HitObjects.OfType<BananaShower>().Select(b => new Period(b.StartTime - TRANSITION_DURATION, b.EndTime)));
```

Comment:
I'm actually not sure whether I'd hide the catcher during showers. They're kind of like normal fruit and also will provide *some* visual feedback, won't they? Also controlling the banana capture path to maximise caught fruit is a thing in catch I think, so it's not like a spinner where you go as fast as possible.